### PR TITLE
Remove firefox extensions

### PR DESCRIPTION
**Pull request by koolhand**
Hi @bcurran, I did some reading this morning,

The "sideloading" method of installing at the machine-level just isn't supported anymore. Mozilla plugins are kinda meant to install into user profile, not at the machine / program-files level. And they self-update using Firefox's own mechanism, in user-land.

You could try switching to use Firefox's [https://github.com/mozilla/policy-templates#extensions](Group Policy reg keys) which forcibly install XPIs, which are intended for a corporate environment where lots of machines are kept in the same config. That would force it to install into every logged on user's profile, separately. But... really you'd be working "against the grain" of what Firefox natively supports.

So bottom line: I think you should remove the *-firefox extension packages from your repo.

This pull request will do that if you're interested.

### Removed:

* colorfultabs-firefox
* flagfox-firefox
* gmailnotifier-firefox
* google-shortcuts-firefox
* https-everywhere-firefox
* ietab2-firefox
* trafficlight-firefox
* wot-firefox

### CODE REVIEW

**Comment by bcurran3** [comment]
I appreciate the effort but I'm not in the habit of removing dead packages at the moment. (There are unrelated others!)
